### PR TITLE
Flagging in live feed

### PR DESCRIPTION
**Pull request by yattias**
- Adding the ability to flag content through the live feed to help with moderation
- Fix "empty box" in paper page when license is "closed access"

image

image

image

### CODE REVIEW

File: components/LiveFeed/Contribution/ContributionHeader.tsx [hunk 1]
```diff
@@ -199,31 +205,137 @@ const ContributionHeader = ({ entry }: Args) => {
...
+            }}
+          >
+            <GenericMenu
+              softHide={true}
+              options={[
+                {
+                  preventDefault: true,
+                  value: "flag",
+                  html: (
+                    <FlagButtonV2
```
Comment (yattias) [inline components/LiveFeed/Contribution/ContributionHeader.tsx:245]: Kinda ugly but required for usage of this component

File: components/LiveFeed/Contribution/ContributionHeader.tsx [hunk 2]
```diff
@@ -199,31 +205,137 @@ const ContributionHeader = ({ entry }: Args) => {
...
-                </ALink>
-              </span>
+              <>
+                <span className={css(styles.dot)}>•</span>
+                <span className={css(styles.unifiedDocument)}>
+                  <ALink
+                    overrideStyle={styles.link}
+                    href={getUrlToUniDoc(unifiedDocument)}
+                  >
+                    {truncateText(unifiedDocument?.document?.title, 100)}
```
Comment (manveerxyz) [inline components/LiveFeed/Contribution/ContributionHeader.tsx:224]: not super opinionated here, but 100 characters seems like a lot, and causes the title to wrap on my 13" MacBook:

Screenshot 2023-11-29 at 2 29 34 PM

**Review by manveerxyz — COMMENTED** [review]
Overall looks good, just a couple of small UI/UX things I noticed:

### Menu button wraps on mobile
Screenshot 2023-11-29 at 2 24 32 PM

### I'm able to open-up multiple popovers
Screenshot 2023-11-29 at 2 24 45 PM

### Clicking on empty-space on button doesn't do anything

https://github.com/ResearchHub/researchhub-web/assets/16143968/1437bd0e-5b69-4617-9764-776a67eb625d

File: components/LiveFeed/Contribution/ContributionHeader.tsx (hunk 2 above)
Comment (yattias) [reply components/LiveFeed/Contribution/ContributionHeader.tsx:224]: Good point, thanks!

**Pull request merged**